website/www/site/layouts/index.html

```diff
@@ -60,6 +60,24 @@ <h2 class="pillars-title">
   </div>
 {{ end }}
 
+{{ define "playground-section" }}
+<div class="body__contained body__section-nav">
+  <h1>Try Beam Playground</h1>
+  <p>Beam Playground is an interactive environment to try out Beam transforms and examples without having to install Apache Beam in your environment.
+    You can try an Apache Beam examples at <a href="https://frontend-beta-dot-apache-beam-testing.appspot.com/">Beam Playground (Beta)</a>.
+  </p>
+  <br>
+  <br>
+  <iframe
+          src="https://frontend-beta-dot-apache-beam-testing.appspot.com/embedded?enabled=true&example=SDK_JAVA/PRECOMPILED_OBJECT_TYPE_EXAMPLE/MinimalWordCount&code="
```

Comment:
can we also use the play.beam.apache.org URL?

Reply:
changed